ticTacToe/src/components/Square.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../App.css';
+
+
```

Comment:
I like how you experimented with this way of styling! I normally just import a css stylesheet, so it's nice to see it a different way and learn from you!